internal/pkg/manager/spod/bindata/spod.go

```diff
@@ -131,12 +132,19 @@ var Manifest = &appsv1.DaemonSet{
 						chown 65535:0 /etc/selinux.d
 						chmod 750 /etc/selinux.d
 						cp /usr/share/udica/templates/* /etc/selinux.d
+						cp /opt/seccomp-profiles/*.cil /etc/selinux.d
+						semodule -i /etc/selinux.d/*.cil
 					`},
 						VolumeMounts: []v1.VolumeMount{
 							{
 								Name:      "selinux-drop-dir",
 								MountPath: SelinuxDropDirectory,
 							},
+							{
+								Name:      "operator-seccomp-profile-volume",
+								MountPath: "/opt/seccomp-profiles",
```

Comment:
We also add selinux policies into this directory, so I guess `seccomp-profiles` is misleading.